README.md

```diff
@@ -118,8 +118,7 @@ newrelic-response-size        10s
 newrelic-span                 10s
 ```
 
-You should start to see metrics sent to [Insights](https://insights.newrelic.com) a few minutes after the deployment.
-As an example, this Insights query will display a timeseries graph of total Istio requests:
+You should be able to query your data via [NRQL](https://docs.newrelic.com/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql) within a few minutes of deployment. For example, this NRQL query will display a timeseries graph of total Istio requests:
```

Comment:
I'm a fan of wrapping on sentence breaks to make reviews of the Markdown changes easier to understand.

```suggestion
You should be able to query your data via [NRQL](https://docs.newrelic.com/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql) within a few minutes of deployment.
For example, this NRQL query will display a timeseries graph of total Istio requests:
```